workshops/guided-projects/guided_project_1.ipynb

```diff
@@ -4,14 +4,24 @@
    "cell_type": "markdown",
```

Comment:
<p>Nit: Should you still have a license added at the end of each notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='93'/>

Reply:
<p>Done! (Thanks good catch!)</p>